pyproject.toml

```diff
@@ -5,7 +5,7 @@ description = "A bridge of aave staticATokens for reward collection on L2"
 authors = ["AAVE", "nethermind"]
 
 [tool.poetry.dependencies]
-python = "3.7.12"
+python = "3.7.13"
```

Comment:
why do we need to upgrade python here?

Reply:
the ci suddenly refused to run with an older version see: https://github.com/aave-starknet-project/aave-starknet-bridge/runs/8204649309?check_suite_focus=true

Reply:
wow, really weird. it means that packages are not perfectly locked, no?

Reply:
anyway, ok to upgrade it if it works well with python 3.7.12. don't forget to also change README.md that mentions Python version to use